server/controllers/filter/getFilter.js

```diff
@@ -1,3 +1,12 @@
-module.exports = (req, res) => {
-  res.send('get filter route');
+const { getFilterData } = require('../../database/queries/filter/index');
+
+module.exports = (req, res, next) => {
+  getFilterData(req.params.member_id)
+    .then((response) => {
```

Comment:
@Angham116 maybe it'd be better if we parse the skills and offer_types here :thinking:
what do you think?